spec/factories/case_group_memberships.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :case_group_membership do
+    case_group { create(:case_group) }
+    casa_case { create(:casa_case) }
```

Comment:
You can simplify this and just call the association, and it will automatically create the association:

```
  factory :case_group_membership do
    case_group
    casa_case
  end
```